client/src/app.tsx

```diff
@@ -124,11 +154,38 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
   if (networkError) console.log(`[Network error]: ${networkError}`);
 });
 
+const afterwareLink = onError(({ graphQLErrors, operation, forward }) => {
+  if (graphQLErrors) {
+    for (const err of graphQLErrors) {
+      if (err.message.includes('token is invalid')) {
```

Comment:
can't we have a ststusCode instead ?

Reply:
I wish! But I didn't find out how... Do you know how we could do that?